convai/conversation_gateways.py

```diff
@@ -612,9 +626,12 @@ def add_messengers(self, *messengers: AbstractMessenger):
         user = await self._update_user_record_in_db(receiving_peer)
         messenger = self._messenger_for_user(user)
         conv = self._conversations[user]
-        external_id = await messenger.send_message_to_user(receiving_peer, msg_text, True)
+        external_id = await messenger.send_message_to_user(receiving_peer, msg_text, self.evaluate_message)

```

Comment:
We should let user to evaluate messages even if `evaluete_message` in config is `False`